x-pack/plugins/security/public/session/session_timeout.ts

```diff
@@ -77,11 +76,8 @@ export class SessionTimeout {
     // Subscribe to a broadcast channel for session timeout messages.
     // This allows us to synchronize the UX across tabs and avoid repetitive API calls.
     try {
-      const { BroadcastChannel } = await import('broadcast-channel');
-      this.channel = new BroadcastChannel(`${this.tenant}/session_timeout`, {
-        webWorkerSupport: false,
-      });
-      this.channel.onmessage = this.handleChannelMessage;
+      this.channel = new BroadcastChannel(`${this.tenant}/session_timeout`);
+      this.channel.onmessage = (event) => this.handleChannelMessage(event);
     } catch (error) {
       // eslint-disable-next-line no-console
       console.warn(
```

Comment:
We originally added this try/catch to prevent everything from blowing up if this async chunk failed to load. Now that there's no async chunk being loaded, I'm not sure we need it now...

I assume the try/catch will allow _slightly_ older browsers that don't support BroadcastChannel (such as Safari 15.3) to continue to work with Kibana, but I don't have such a browser to test with. I guess it doesn't hurt to leave the try/catch here, in any case!